mmeds/tests/unit/test.py

```diff
@@ -0,0 +1,144 @@
+from subprocess import run
+import sys
+import os
+
+from mmeds.authentication import add_user, remove_user
+from mmeds.database import upload_metadata, upload_otu
+
+import mmeds.config as fig
+import mmeds.secrets as sec
+
+'''
+- to run all the tests: python test.py
+- to run a specific set of test: python test.py test_name1 test_name2 etc
+  - possible test names: authentication, database, documents, spawn, tool, tools, util, validate
+- to run all tests with error log output: python test.py log
+  - gives output as wanted for Travis CI
+'''
+
+testing = True
+log = False
+
+def main():
+    global log
+    if 'log' in sys.argv:
+        log = True
```

Comment:
if you want to be fancy you can do `log = 'log' in sys.argv` since the expression evaluates to `True` or `False`